src/metrics/scenario.jl

```diff
@@ -279,3 +282,72 @@ function scenario_outcomes(rs::ResultSet, metrics::Vector{<:Metric})::YAXArray
 
     return scen_outcomes
 end
+
+"""
+    cf_difference_scenario(rs::ResultSet, metric::Function)
+    cf_difference_scenario(scens::DataFrame, outcomes::Union{YAXArray{Float64,3},
+        YAXArray{Float64,2},YAXArray{Float32,3},YAXArray{Float32,2}})
+    cf_difference_scenario(
+        scens::DataFrame, outcomes::Union{YAXArray{Float64,1},YAXArray{Float32,1}})
+
+Get bootstrapped difference between each intervention and conterfactual scenarios.
+
+# Arguments
+- `rs` : ResultSet
+- `metric` : Function to use to calculate metric.
+- `scens` : Scenario DataFrame
+`outcomes` : YAXArray for a metric (can have 1-3 dimensions which must include :scenarios)
+
+# Returns
+YAXArray with (:timesteps, :scenarios, :outcomes)
+
+# Examples
+```
+metrics::Vector{ADRIA.metrics.Metric} = [
+    ADRIA.metrics.scenario_total_cover,
+    ADRIA.metrics.scenario_asv,
+    ADRIA.metrics.scenario_absolute_juveniles,
+]
+
+# 3-dimensional Array of outcomes
+outcomes = ADRIA.metrics.scenario_outcomes(rs, metrics)
+```
+"""
+function cf_difference_scenario(rs::ResultSet, metric::Function)
```

Comment:
This was previously added because the economics team use the difference between counterfactuals and intervention scenarios in their DEA analysis. But I think I'll remove for now and add in a later PR to better integrate with your bootstrapping functions (yours may even be able to do this already with a few tweaks)